### ISSUE TITLE
Using show() with multiple renderers

### ISSUE THREAD

**Issue by medakk** [marcomusy/vedo#315]
I have a plotter initialized as:

```python
vp1 = vedo.Plotter(shape=(3,5), axes=0)
```

I've added meshes and spheres to these and they display fine. Now, I have a loop where I'm updating the positions of these spheres. At the end of the update, I want to redraw all the renderers. If I try:

```python
vp1.show(interactive=False, at=0, resetcam=False)
```
the camera control(by mouse) works, but the 0th renderer has ALL the spheres in(ie the ones I created in other renderers):

![image](https://user-images.githubusercontent.com/10536745/107893378-104ecd80-6ee8-11eb-888c-1c9a1900f92f.png)

Instead, if I update the show command to:

```python
vp1.show()
```

Now the number of spheres is correct, but I cannot move the camera.
![image](https://user-images.githubusercontent.com/10536745/107893430-5c9a0d80-6ee8-11eb-8e0c-e645e01e0aee.png)

I'm running the latest dev build(was trying to follow the examples but it seems some of them conform to the latest build, and some to an older version of the library). Here is the complete source code:

```python
import vedo
from random import random

# this is one instance of the class Plotter with 3 rows and 5 columns
vp1 = vedo.Plotter(shape=(3,5), axes=0)

for i in range(15):
    vp1.show(str(i),at=i)

class Particle:
    def __init__(self, at):
        global vp1
        self.p = vedo.vector((0.0, 0.0, 0.0))
        self.vsphere = vedo.Sphere(self.p, r=0.1, c='red')
        vp1.add(self.vsphere, at=at)
    
    def update(self):
        self.vsphere.pos(self.p)


particles = []
for d in range(15):
    for i in range(5):
        particles.append(Particle(at=d))

while True:
    for particle in particles:
        particle.p[0] += (random()-0.5)*0.2
        particle.update()

    # I can control the camera fine, but the 0-th renderer has ALL the particles, not
    # just the ones that were added to it.
    # vp1.show(interactive=False, at=0, resetcam=False)

    # This version will render correctly, but I cannot control the camera with the mouse
    vp1.show()
```

Basically, I want to update the positions of spheres in all the renderers, while continuing to be able to interactively adjust the view with the mouse. How do I accomplish this?

**Comment by marcomusy** [comment]
Thanks @medakk for the detailed explanation of the issue. After playing a bit with it I 've come up with this:
```python
import vedo
from random import random

vp1 = vedo.Plotter(shape=(3,5), size=(2000,500))

axe = vedo.Axes(xrange=(-1,1), yrange=(-.2,.2))

for i in range(15):
    vp1.show(str(i), axe, at=i, zoom=2)

class Particle:
    def __init__(self, at):
        self.p = vedo.vector((0, 0, 0))
        self.vsphere = vedo.Sphere(self.p, r=0.05, c='red3').lighting('off')
        vp1.add(self.vsphere, at=at)

    def update(self):
        self.vsphere.pos(self.p)


particles = []
for d in range(15):
    for i in range(5):
        particles.append(Particle(at=d))

# vd = vedo.Video()
for i in range(100):

    for particle in particles:
        particle.p[0] += (random()-0.5)*0.1
        particle.update()

    vp1.allowInteraction()
    vp1.render()
    # vd.addFrame()

# vd.close()
vedo.interactive()
```
![ezgif com-gif-maker](https://user-images.githubusercontent.com/32848391/107959846-320d8a80-6fa4-11eb-80b0-10a99d420f15.gif)
which allows for mouse interaction.

**Comment by medakk** [comment]
That's perfect, thank you! :)

**Comment by maghamis** [comment]
Why [this code](https://github.com/marcomusy/vedo/issues/315#issuecomment-779265775) doesn't work for one renderer? I tried shape(1,1) and N=1 in the plotter but it doesn't work.

**Comment by marcomusy** [comment]
Your link points to this page..
No need to put anything if you have one renderer (remove both keywords).

**Comment by maghamis** [comment]
Thank you for your reply. This is what I am trying and nothing shows up in the renderer:

`import vedo
from random import random

vp1 = vedo.Plotter(size=(2000,500))

axe = vedo.Axes(xrange=(-1,1), yrange=(-.2,.2))

vp1.show("Viewer", axe, zoom=2)

class Particle:
    def __init__(self):
        self.p = vedo.vector((0, 0, 0))
        self.vsphere = vedo.Sphere(self.p, r=0.05, c='red').lighting('off')
        vp1.add(self.vsphere)

    def update(self):
        self.vsphere.pos(self.p)


particles = []

for i in range(5):
    particles.append(Particle())

for i in range(100):

    for particle in particles:
        particle.p[0] += (random()-0.5)*0.1
        particle.update()

    vp1.allowInteraction()
    vp1.render()

vedo.interactive()`

I need to do a similar thing, insert a few points and update their position in a loop. Is there any other example that does the same?

**Comment by medakk** [comment]
@maghamis try the particle simulator example: https://github.com/marcomusy/vedo/blob/master/examples/simulations/particle_simulator.py

**Comment by marcomusy** [comment]
maybe you just missed:
`vp1 = vedo.Plotter(size=(2000,500), interactive=False)`

otherwise your script works fine on my pc..

**Comment by maghamis** [comment]
`interactive=False` solved my problem. Thank you very much!
